static-site/components/list-resources/types.ts

```diff
@@ -16,13 +16,30 @@ export interface Group {
   resources: Resource[];
   groupUrl?: string;
   groupDisplayName?: string;
+  fetchHistory?: FetchGroupHistory;
 }
 
+/* helper function / type guard */
+export function isGroup(x: Group|Resource): x is Group {
+  return Object.hasOwn(x, 'resources');
+}
+
```

Comment:
Strong, non-blocking opinion: `types.ts` files should not export functions, only types or interfaces.